post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -29,7 +25,11 @@ When the operand removes the scheduling gate, the pod enters the scheduling cycl
 
 [IMPORTANT]
 ====
-The Technology Preview version of the Multiarch Tuning Operator does not support clusters with restricted network scenarios.
+If you are using node affinity and node selectors in the same pod configuration, note the following operand behaviors:
```

Comment:
```suggestion
```

It doesn't matter if users are using one, the other or both.

Reply:
How about updating it to just "Note the following operand behaviors: